src/components/stage/stage.jsx

```diff
@@ -57,13 +64,43 @@ const StageComponent = props => {
                 />
             ) : null}
         </div>
+    ) : (
+        <div>
+            <Box className={styles.stageWrapperOverlay}>
+                <Box
+                    className={classNames(styles.stage, styles.stageOverlayContent)}
+                    componentRef={canvasRef}
+                    element="canvas"
+                    height={heightCorrectedAspect}
+                    width={widthCorrectedAspect}
+                    {...boxProps}
+                />
+                <Box className={styles.monitorWrapper}>
+                    <MonitorList />
+                </Box>
+                <div className={styles.stageOverlayContent}>
+                    <div
+                        className={styles.questionWrapper}
+                        style={{width: widthCorrectedAspect}}
+                    >
+                        {question === null ? null : (
+                            <Question
+                                question={question}
+                                onQuestionAnswered={onQuestionAnswered}
+                            />
+                        )}
+                    </div>
+                </div>
+            </Box>
+        </div>
```

Comment:
This duplication here is asking for divergence. It's hard to tell exactly what you changed — can this be rewritten to just update what you need based on whether `isZoomed` is true or not?

E.g., one difference I see is
```jsx
            <Box
                className={classNames(styles.stageWrapper, {
                    [styles.withColorPicker]: isColorPicking
                })}
            >
// to
              <Box className={styles.stageWrapperOverlay}>
```

This can be rewritten without duplication as:
```jsx
            <Box
                className={classNames({
                    [styles.stageWrapper]: !isZoomed,
                    [styles.stageWrapperOverlay]: isZoomed,
                    [styles.withColorPicker]: !isZoomed && isColorPicking
                })}
            >
```

Reply:
I took your suggestion here and applied it for the rest of the component to get rid of the duplication. I still needed a double wrapper around the question component to force its width in zoom mode. I'm not sure if there is a better way for that sort of thing.

I also noticed I was cutting off the bottom border when zoomed and since I was touching this code to fix the above I added a margin when zoomed and adjusted the height for this.